recipes/default.rb

```diff
@@ -1,16 +1,12 @@
-package 'apache2' do
-  package_name value_for_platform_family(:rhel => "httpd", :debian => "apache2")
+package 'a better top' do
+  package_name value_for_platform_family(:rhel => "htop", :debian => "atop")
   retries 3
   retry_delay 5
   action :install
 end
-package 'git' do
-  action :/tmp/install
-end
-ruby_block "ensure only our unicorn version is installed by deinstalling any other version" do
```

Comment:
**Do not create recipes that you do not know what it is doing in intricate details**. Your first recipes should be simple ones, that you're doing something that is achieving what you want out of it. Simple one off task.

You are trying to install `git`. Git is already installed.